ha/setup/pre_disruptive_upgrade.py

```diff
@@ -123,17 +131,40 @@ def delete_resources() -> None:
         UpgradeError
     """
     try:
-        root = _get_cib_xml()
-        resources = [e.attrib["id"] for e in root.findall(".//lrm_resource")
-                     if "id" in e.attrib]
+        resources = _get_resource_list()
         Log.info(f"Going to delete following resources: {resources}")
-
         for r in resources:
-            Log.info(f"Deleting {r}")
-            SimpleCommand().run_cmd(f"pcs resource delete {r}")
+            Log.info(f"Deleting resource {r}")
+            SimpleCommand().run_cmd(f"pcs resource delete {r} --force")
```

Comment:
I know this is existing, but can we add "pcs resource delete {r} --force" to const file?

Reply:
Updated